graylog2-web-interface/src/components/datanode/migrations/MigrationWelcomeStep.tsx

```diff
@@ -44,58 +47,72 @@ const StyledHelpPanel = styled(StyledPanel)`
-            This is the recommended method if you want to quickly migrate to data node.
-          </p>
-          <p>
-            If you want to selectively migrate data (e.g. if you use your search backend non-exclusively for Graylog),
-            you should choose the <code>remote reindexing migration</code>.
-            In this scenario, all data will be copied from your existing search backend to data node’s OpenSearch.
-            Depending on your setup, this can take some time and imposes additional disk space for the copied data to be available.
-            During the remote reindexing, Graylog is ingesting data into data node and can be used,
-            but will only serve the data from the old search backend as it becomes available.
-          </p>
-          <p>
-            If you are running <code>ElasticSearch</code> as your search backend <code>remote reindexing migration</code> will automatically be chosen.
-          </p>
-          <p>
-            If you don’t plan to migrate any existing data or only want to migrate a small subset of data
-            we recommend you choose the remote reindexing migration and either skip the data migration or choose only the selected indices for migration.
-          </p>
-        </Panel.Body>
-      </StyledHelpPanel>
-    </Col>
-  </Row>
-);
+const MigrationWelcomeStep = ({ currentStep, onTriggerStep, hideActions }: MigrationStepComponentProps) => {
+  const isElasticsearch = useIsElasticsearch();
+  const isRemoteReindexingEnabled = AppConfig.isFeatureEnabled('remote_reindex_migration');
+  
+  return (
+    <Row>
+      <Col md={isRemoteReindexingEnabled ? 6 : 12}>
+        {isElasticsearch && !isRemoteReindexingEnabled && (
+          <Alert bsStyle="warning">
+            Incompatible search backend. Please upgrade to OpenSearch to be able to use the migration wizard.
+          </Alert>
+        )}
+        <MigrationError errorMessage={currentStep.error_message} />
+        <Headline>Data Nodes Migration</Headline>
+        <p>
+          The Graylog Data Node is a management component designed to configure and optimize OpenSearch for use with Graylog, reducing administrative overhead and simplifying future updates.
+        </p>
+        <p>Deployments earlier than v5.2 or that opted to not install with a Data Node will need to migrate the message databases to Data Nodes.</p>
+        <p>
+          This migration tool will check the compatibility of your components and guide you through to migrate your existing OpenSearch data to a Data Node.<br />
+        </p>
+        <p>Migrating to Data Node will require some steps to be performed on the OS, within your current OS/ES cluster, and in your configuration files.</p>
+        <p>You can get more information on the Data Node migration <DocumentationLink page="graylog-data-node" text="documentation" />.</p>
+        <br />
+        <MigrationDatanodeList />
+        {!(isElasticsearch && !isRemoteReindexingEnabled) && (
```

Comment:
```suggestion
        {(!isElasticsearch && !isRemoteReindexingEnabled) && (
```
makes it clearer I think

Reply:
that's not a typo it's the negation of (isElasticsearch && !isRemoteReindexingEnabled)

Reply:
@grotlue it looks like that changes the logic